### ISSUE TITLE
spkg-configure.m4 for maxima

### ISSUE THREAD

**Issue by orlitzky** [sagemath/sage#32867]
<div id="comment:0"></div>

We handled ECL by itself in #29617, so now maxima needs its own ticket.

This would actually be "easy" at this point if not for our custom `matrixexp.patch` (https://sourceforge.net/p/maxima/bugs/2596/).

Depends on #33718

CC:  @slel @antonio-rojas

Component: **build: configure**

Keywords: **maxima**

Author: **Michael Orlitzky**

Branch/Commit: **[u/mjo/ticket/32867](https://github.com/sagemath/sagetrac-mirror/tree/u/mjo/ticket/32867) @ [`544b601`](https://github.com/sagemath/sagetrac-mirror/commit/544b6014e4b88aba1d5c9b569878db95de184466)**

Reviewer: **https://github.com/sagemath/sage/actions/runs/1992568379**

_Issue created by migration from https://trac.sagemath.org/ticket/32867_

**Comment by dimpase** [comment]
done in #35615
